File: db4o.net/Db4objects.Db4o/Db4objects.Db4o/Internal/Handlers/Array/ArrayHandler5.cs
/* Copyright (C) 2004 - 2011  Versant Inc.  http://www.db4o.com */

using Db4objects.Db4o.Internal.Handlers.Array;

namespace Db4objects.Db4o.Internal.Handlers.Array
{
	/// <exclude></exclude>
	public class ArrayHandler5 : ArrayHandler
	{
		protected override ArrayVersionHelper CreateVersionHelper()
		{
			return new ArrayVersionHelper5();
		}
	}
}

File: db4oj/core/src/com/db4o/internal/handlers/array/ArrayHandler5.java
/* Copyright (C) 2009  Versant Inc.  http://www.db4o.com */

package com.db4o.internal.handlers.array;

/**
 * @exclude
 */
public class ArrayHandler5 extends ArrayHandler {
	
    protected ArrayVersionHelper createVersionHelper() {
        return new ArrayVersionHelper5();
    }

}
